crates/biome_analyze/src/context.rs

```diff
@@ -97,6 +100,10 @@ where
     pub fn is_global(&self, text: &str) -> bool {
         self.globals.contains(&text)
     }
+    /// Checks whether the specifier is a dependency
+    pub fn is_dependency(&self, specifier: &str) -> bool {
+        self.dependencies.contains(&specifier)
+    }
```

Comment:
This seems too specific to me. I think we should expose the whole package.json configuration instead. This might allow more rules to be written.

Reply:
This means that `biome_analyze` will have a dependency on `biome_project`, which I would like to avoid. I suppose I'll have to create a service for that.